src/app/results/results.component.ts

```diff
@@ -76,7 +76,7 @@ export class ResultsComponent implements OnInit {
 
   filterByDate() {
     let urldata = Object.assign({}, this.searchdata);
-    urldata.query += urldata.query.substr(urldata.query.length - 6, 6) !== " /date" ? " /date" : "";
+    // urldata.query += urldata.query.substr(urldata.query.length - 6, 6) !== " /date" ? " /date" : "";
```

Comment:
Please leave it as it is.,since the filterByDate() method is already commented in the component.html file. It might create ambiguity in code reading afterwards. Rest looks fine :+1: .

Reply:
And do not forget to squash your commits.